modules/va_facilities/README.yml

```diff
@@ -837,6 +840,9 @@ components:
               enum:
                 - "A"
                 - "T"
+            visn:
+              type: string
+              example: '20'
```

Comment:
Incredibly petty, but it looks like the rest of the examples in this schema object have double quotes.